src/pages/Lend/LendHeader.vue

```diff
@@ -17,6 +17,16 @@ import ViewToggle from '@/components/LoansByCategory/ViewToggle';
 export default {
 	components: {
 		ViewToggle,
+	},
+	props: {
+		browseUrl: {
+			type: String,
+			default: '/lend-by-category'
```

Comment:
I think these defaults can prob be empty at this level

Reply:
@mcstover Will update to blank/empty.